channels/middleware/authorization.go

```diff
@@ -498,6 +577,13 @@ func (am *authorizationMiddleware) RemoveParentGroup(ctx context.Context, sessio
 		}); err != nil {
 			return errors.Wrap(err, errGroupRemoveChildChannels)
 		}
+		params := map[string]interface{}{
+			"entity_id":       id,
+			"parent_group_id": ch.ParentGroup,
+		}
+		if err := am.Callout(ctx, session, channels.OpRemoveParentGroup.String(channels.OperationNames), params); err != nil {
```

Comment:
@dborovcanin @nyagamunene
This is not the right approach.
`channels.OpRemoveParentGroup.String(channels.OperationNames)`
For now lets with this , later we can change to cleaner approach I have create cleaner approach ticket here https://github.com/absmach/supermq/issues/2884